packages/embed-react/src/components/widget.tsx

```diff
@@ -24,7 +24,7 @@ export const Widget = ({ id, style = {}, className = '', ...props }: WidgetProps
   return (
     <>
       <InlineStyle filename="widget" />
-      <div style={style} className={className} ref={container} />
+      <div style={style} className={className} ref={container} data-testid="tf-v1-widget-container" />
```

Comment:
yes is there, but I need the element with the styles as well for the tests

Reply:
How about `document.querySelector('.tf-v1-widget').parentElement`? :) I would like to avoid adding extra attributes just for the purpose of testing, especially if this is added to our customer websites.

Reply:
yeah but if the component structure changes then the test fails, with `data-testid` doesn't matter at all